vnext/template/metro.devMode.config.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Metro configuration for React Native
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+const fs = require('fs');
+const path = require('path');
+const rnwPath = fs.realpathSync(
+  path.resolve(require.resolve('react-native-windows/package.json'), '..'),
+);
+const hoistedModules = fs.realpathSync(
+  path.resolve(rnwPath, '..', 'node_modules'),
+);
+module.exports = {
+  watchFolders: [rnwPath, hoistedModules],
+  resolver: {
+    extraNodeModules: {
+      // Redirect react-native-windows to avoid symlink (metro doesn't like symlinks)
+      'react-native-windows': rnwPath,
+    },
+  },
+  transformer: {
```

Comment:
Maybe add a `//CodeSync: metrho.config.js` comment to indicate that things hve to stay in sycn between this and the regular one.